src/Servers.jl

```diff
@@ -412,6 +418,9 @@ function handle_transaction(f, t::Transaction, server; final_transaction::Bool=f
         closeread(http)
         @debug 2 "server closewrite"
         closewrite(http)
+        if server.access_log !== nothing
+            @info sprint(server.access_log, http) _group=:access_log
```

Comment:
might want to call the group just `:access` ?
Saying `:access_log` seems redudant.

Reply:
Sure, updated.